src/App.js

```diff
@@ -8,29 +8,35 @@ import Feed from "./components/feed/Feed";
 import AllPropertiesData from "./components/allproperties/AllPropertiesData";
 import Profile from "./pages/profile/Profile";
 import Sales from "./components/sales/Sales";
-import Rent from "./components/rent/Rent"
-
-
-
+import Rent from "./components/rent/Rent";
+import PropertyDetails from "./components/propertydetails/PropertyDetails";
+import Description from "./components/propertydetails/Description";
+import Review from "./components/propertydetails/Review";
+import Features from "./components/propertydetails/Features";
 
 function App() {
   return (
     <div className="App">
       <Router>
         <Routes>
           <Route path="/" element={<Homepage />}></Route>
-          <Route path="/properties" element={<AllPropertiesData/>}></Route>
+          <Route path="/properties" element={<AllPropertiesData />}></Route>
           <Route path="/signup" element={<Signup />}></Route>
           <Route path="/login" element={<Login />}></Route>
           <Route path="/dashboard" element={<Dashboard />}></Route>
-          <Route path="/my-properties" element={<Feed/>}></Route>
-          <Route path="/post-a-property" element={<PropertiesInputData/>}></Route>
-          <Route path="/profile" element={<Profile/> }></Route>
-          <Route path="/sales" element={<Sales/>}></Route>
-          <Route path="/rent" element={<Rent/>}></Route>
-     
-      
-          
+          <Route path="/my-properties" element={<Feed />}></Route>
+          <Route
+            path="/post-a-property"
```

Comment:
I guess the path name should be review. eg post-property

Reply:
okay.